Sources/Build/BuildParameters.swift

```diff
@@ -88,6 +93,7 @@ public struct BuildParameters: Encodable {
         destinationTriple: Triple = Triple.hostTriple,
         flags: BuildFlags,
         toolsVersion: ToolsVersion = ToolsVersion.currentToolsVersion,
+        jobs: UInt32 = UInt32(ProcessInfo.processInfo.activeProcessorCount),
```

Comment:
Updated to make this initializer parameter optional, so that sourcekit-lsp or other projects invoking this function don't have to be changed.